### ISSUE TITLE
Thank you for your trust

### ISSUE THREAD

**Issue by ciscoheat** [bje259/WineTrackerSkv2#2]
Hey, I stumbled upon this repo, just wanted to say thank you for putting your trust in all of my libraries. :)

**Comment by bje259** [comment]
Absolutely!  Than you for all your time an effort creating tools for the rest of us!  I haven't dove into Superforms 2.0 yet, but I can already tell you've been putting a ton of work into it :).
